exo-go/src/app_setup/helpers.go

```diff
@@ -0,0 +1,30 @@
+package appSetup
+
+import (
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/Originate/exosphere/exo-go/test_helpers"
+)
+
+// CheckoutApp copies the example app appName to cwd
+func CheckoutApp(cwd, appName string) error {
+	src := path.Join("..", "..", "..", "exosphere-shared", "example-apps", appName)
+	dest := path.Join(cwd, appName)
+	err := os.RemoveAll(dest)
+	if err != nil {
+		return err
+	}
+	return testHelpers.CopyDir(src, dest)
+}
```

Comment:
As I mentioned, we can't use the existing function in `test_helpers/index.go` because of the path difference. I think we might need to pass in relative path.